packages/api/roles/admin.js

```diff
@@ -31,8 +31,11 @@ export default (role, actions) => {
   role.allow(actions.updateOrder, () => true);
   role.allow(actions.updateOrderDiscount, () => true);
   role.allow(actions.updateOrderItem, () => true);
-  role.allow(actions.updateOrderDelivery, () => true);
```

Comment:
While I think it did make sense to duplicate the resolvers so we have a file per query/mutation this is not necessarily true for the actions. I can't think of any case where you want to allow a user to update the context of a pickup delivery but disallow updating the context of a shipping delivery